src/index.js

```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import { legacy_createStore as createStore } from 'redux';
+import { Provider } from 'react-redux';
 import './index.css';
 import { BrowserRouter as Router } from 'react-router-dom';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import rootReducer from './redux/configureStore';
+
+const store = createStore(rootReducer);
```

Comment:
- [ ] [OPTIONAL] Kindly move this line to the `configureStore.js` file and import the created store instead of creating it here since this is a general convention used to create a redux store.